native/screens/AddPortfolioProjects.jsx

```diff
@@ -0,0 +1,92 @@
+import { ScrollView, Alert } from 'react-native';
```

Comment:
we haven't talked too much about this convention but i think a healthy way of organizing so many imports is by stacking them like:
- packages, libraries (uuid, useSelector form react-redux, etc) 
- components 
- assets (style sheets, homemade utility functions)